Automattic/wp-calypso
# BackupBrowser: Create theme/plugin/table/files lists in confirmation page

## Proposed Changes

* Populates the confirmation page with theme, plugin, table and file separated lists
* Each section has a maximum of 10 item defaultly visible
* The category can be expanded to show all and the entire category can be collapsed
* When collapsed the category will revert back to showing only 10 when re-expanded
* If all themes, plugins, or tables are selected a message stating that is displayed
* The Confirm Restore button will restore the files

## Testing Instructions



* Load up the backup browser appending `?flags=jetpack/backup-granular` to the URL if using a calypso live link
* Use the select option to select multiple files and types of files
* Select Restore from the select header
* Ensure the files appear as expected in the proper sections
* Ensure the sections behave as described
* Test the Confirm Restore button functions properly

## Pre-merge Checklist




## Translation Image References
<img width="709" alt="Screenshot 2023-10-05 at 6 28 47 PM" src="https://github.com/Automattic/wp-calypso/assets/1992658/a4a58f96-94cc-42a3-ae68-490add6b7ff4">
<img width="722" alt="Screenshot 2023-10-05 at 6 29 13 PM" src="https://github.com/Automattic/wp-calypso/assets/1992658/9a42957d-ca2f-4d63-806c-db1dee29ad9e">
